WORKSPACE

```diff
@@ -884,7 +884,7 @@ register_toolchains("//:nix_python_toolchain") if not is_windows else None
 
 nixpkgs_package(
     name = "postgresql_nix",
-    attribute_path = "postgresql",
+    attribute_path = "postgresql_9_6",
```

Comment:
I’ll leave it to @gerolf-da’s team to decide if and when they want to upgrade. For now, this PR keeps the version at `9.6` which we also used before.